x/wasm/README.md

```diff
@@ -15,14 +15,193 @@ query_gas_limit = 300000
 lru_size = 0
 ```
 
+## Events
+
+A number of events are returned to allow good indexing of the transactions from smart contracts.
+
+Every call to Instantiate or Execute will be tagged with the info on the contract that was executed and who executed it.
+It should look something like this (with different addresses). The module is always `wasm`, and `code_id` is only present
+when Instantiating a contract, so you can subscribe to new instances, it is omitted on Execute:
+
+```json
+{
+    "Type": "message",
+    "Attr": [
+        {
+            "key": "module",
+            "value": "wasm"
+        },
+        {
+            "key": "signer",
+            "value": "cosmos1vx8knpllrj7n963p9ttd80w47kpacrhuts497x"
+        },
+        {
+            "key": "code_id",
+            "value": "1"
+        },
+        {
+            "key": "contract_address",
+            "value": "cosmos18vd8fpwxzck93qlwghaj6arh4p7c5n89uzcee5"
+        }
+    ]
+}```
```

Comment:
Markup is currently broken (https://github.com/cosmwasm/wasmd/blob/cosmwasm-0.7/x/wasm/README.md#events). I guess thus needs a lint break before ``` in order to end the code block.